jvm/core/src/main/scala/org/scalatest/PrivateMethodTester.scala

```diff
@@ -102,10 +102,919 @@ trait PrivateMethodTester {
      * The type parameter, <code>T</code>, is the return type of the private method.
      *
      * @param args zero to many arguments to pass to the private method when invoked
-     * @return an <code>Invocation</code> object that can be passed to <code>invokePrivate</code> to invoke
-     * the private method
+     * @return an <code>Invocation0</code> object that can be passed to <code>invokePrivate</code> to invoke the private method
      */
-    def apply(args: Any*) = new Invocation[T](methodName, args: _*)
+    def apply() = new Invocation0[T](methodName)
+  }
```

Comment:
I think we should leave PrivateMethod alone so that no code breaks. We can decide to deprecate it, but that would give folks time to change to PrivateMethodN. Or we could not deprecate it and just let people choose. So please take PrivateMethod and plain old Invocation back to the way it was, then probably add a PrivateMethod0 trait that would use the Invocation0 object.